airflow/utils/callback_requests.py

```diff
@@ -59,8 +59,10 @@ def __init__(
         simple_task_instance: SimpleTaskInstance,
         is_failure_callback: Optional[bool] = True,
         msg: Optional[str] = None,
+        task: Optional[Any] = None,
```

Comment:
We can't pass the task here -- this is a "message" that goes between processes, and by doing this it will require serialization of the task, but the task here in the scheduler is Serialized and so won't have access to any of the code etc.

Instead (if we need this sort of behaviour) we'll have to load the dag in the processor (which we already do) and _then_ execute it.